### DISCUSSION TITLE
Phantom对本地开发显示不友好

### DISCUSSION THREAD

**Discussion by v1xingyue** [creatorsand/solana-co-learn#10] [Q&A] [answered]
按照 文档设置 phantom 的 开发者网络为: 

![image](https://github.com/CreatorsDAO/all-in-one-solana/assets/974169/e85bf01b-43e0-4fb6-8687-c938034475b7)

然后钱包，显示空白， 如下： 
![image](https://github.com/CreatorsDAO/all-in-one-solana/assets/974169/2138f10b-f3fc-4bd2-a55a-989316d1f14f)

本地 test-validator  启动正常，可以使用 cli 接口获取余额，发布等操作。 

![image](https://github.com/CreatorsDAO/all-in-one-solana/assets/974169/49793a43-940e-4c0d-bfc8-5d5742747f89)

**Comment by DaviRain-Su** [comment, marked as answer]
推荐使用backpack 🎒钱包： https://creatorsdao.github.io/all-in-one-solana/docs/Solana-Co-Learn/module1/wallet-usage/
